packages/web-components/src/components/va-breadcrumbs/va-breadcrumbs-slot.css

```diff
@@ -0,0 +1,66 @@
+va-breadcrumbs .va-breadcrumbs-li {
+  display: inline-block;
+  line-height: 1.5;
+  margin: 0.4rem 0;
+  padding: 0.4rem 0 0.4rem 0;
+  vertical-align: middle;
+}
+
+va-breadcrumbs .va-breadcrumbs-li::after {
+  content: ' › ';
+  display: inline-block;
+  padding: 0 0.56rem;
+}
+
+va-breadcrumbs .va-breadcrumbs-li:last-child::after {
+  content: '';
+}
+
+@media screen and (max-width: 481px) {
+  va-breadcrumbs .va-breadcrumbs-li {
+    display: none;
+  }
+
+  va-breadcrumbs .va-breadcrumbs-li:nth-last-child(2) {
+    display: inline-block;
+  }
+
+  va-breadcrumbs .va-breadcrumbs-li:nth-last-child(2)::before {
+    content: ' \2039 ';
+    display: inline-block;
+    padding: 0 0.24rem;
+  }
+
+  va-breadcrumbs .va-breadcrumbs-li:nth-last-child(2)::after {
+    content: '';
+    display: none;
+    padding: 0;
+  }
+}
+
+va-breadcrumbs li > a {
+  background-color: transparent;
+  color: var(--color-link-default);
+  display: inline-block;
+  padding: 0.2rem;
+  transition-duration: 0.3s;
+  transition-property: color, background-color, border-color;
+  transition-timing-function: ease-in-out;
+}
+
+va-breadcrumbs li > a:hover {
+  background: rgba(0, 0, 0, 0.05);
+}
+
+va-breadcrumbs li > a:focus {
+  outline: 2px solid var(--color-gold-light);
+  outline-offset: 0.2rem;
+}
```

Comment:
This screenshot is from a page in `vets-website` using this web component. Most of the styles are the same with some exceptions.

<img width="397" alt="Screen Shot 2022-02-17 at 19 22 17" src="https://user-images.githubusercontent.com/36863582/154593932-5784469a-22a7-492d-9e93-a2262b64cbfb.png">

I left them in because I don't like depending on external styles. That could lead to our components not displaying properly if styling changes are made elsewhere. Although it may lead to more chore work, I like it because it's more predictable and controllable. It makes our component-library components more self-contained. If styles need to be updated, there will be tickets created for that and it won't catch us by surprise. I would rather work on a ticket that is about updating styling and not about fixing a bug in our components because of external styling changes.

Should our components inherit external styles or should they contain all the styling? @bkjohnson @k80bowman @pdavies88 @mendacitees